src/EndpointManager.sol

```diff
@@ -79,6 +89,24 @@ abstract contract EndpointManager is
         }
     }
 
+    function _getOutboundQueueStorage()
+        internal
+        pure
+        returns (mapping(uint64 => OutboundQueuedTransfer) storage $)
```

Comment:
No particular reason -- I had used `uint64` for sequence number since that's the sequence number type used by the Wormhole token bridge.

Perhaps the token bridge used it for compatibility reasons with other components? Don't know all the history here, cc @kcsongor

Reply:
Adding context here -- this is best for compatibility since on most other chains the native word size is `u64`.